inst/htmljs/animint.js

```diff
@@ -228,6 +228,9 @@ var animint = function (to_select, json_file) {
     g_info.tr.append("td").attr("class", "chunk");
     g_info.tr.append("td").attr("class", "downloaded").text(0);
     g_info.tr.append("td").text(g_info.total);
+    g_info.tr.append("td").attr("class", "total_MB").text("0.00");
+    g_info.tr.append("td").attr("class", "mean_MB").text("0.00");
```

Comment:
in these lines you should save the td so you can update it later withoug having to do select()
```suggestion
var td_mean_MB = g_info.tr.append("td").attr("class", "mean_MB");
```